Example/Messaging/Tests/FIRMessagingRemoteNotificationsProxyTest.m

```diff
@@ -256,6 +262,7 @@ - (void)testSwizzledAppDelegateRemoteNotificationMethods {
   XCTAssertTrue(appDelegate.remoteNotificationWithFetchHandlerWasCalled);
 
   [self.mockMessaging verify];
+#endif
```

Comment:
Can only `application:didReceiveRemoteNotification:fetchCompletionHandler:` be excluded for macOS?

Reply:
Done. I'm not sure why but appDidReceiveMessage: is not swizzled inside macOS. Do you have any thoughts that might cause this?

Reply:
@chliangGoogle Do you mean `appDidReceiveMessage:` is not called for you on macOS when `application:didReceiveRemoteNotification:` is called even when the app delegate is swizzled?
If so, I tried it locally and it seems to be working for me.

Reply:
It's just the unit test failing for me, for the fakeAppDelegate and IncompleteAppDelegate.